src/vc.rs

```diff
@@ -871,13 +883,30 @@ impl Default for Presentation {
 
 /// Get a DID's first verification method
 pub async fn get_verification_method(did: &str, resolver: &dyn DIDResolver) -> Option<String> {
-    let vms = get_verification_methods(did, resolver)
-        .await
-        .unwrap_or_default();
-    vms.into_iter().next()
+    let (res_meta, doc_opt, _meta) = resolver
+        .resolve(did, &ResolutionInputMetadata::default())
+        .await;
+    if res_meta.error.is_some() {
+        return None;
```

Comment:
Do we really want to hide this error? It could make the code a bit messy as we probably don't want to stop whatever operation is in progress because one resolution failed, but I'm afraid the potential error for the end user might be misleading.

Reply:
I agree this is not ideal. Issue opened: #126